e2e/not-found.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
+
+const pageElements = {
+  notFoundImage: 'not-found-image',
+  notFoundHeading: 'not-found-heading',
+  heresQuoteParagraph: 'heres-quote-paragraph',
+  quoteWrapper: 'quote-wrapper',
+  quoteContent: 'quote-content',
+  authorName: 'author-name',
+  viewCurriculumBtn: 'view-curriculum-btn'
+};
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/404');
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
```

Comment:
```suggestion
test.beforeEach(async ({ page }) => {
  await page.goto('/404');
});

test.afterEach(async () => {
  await page.close();
});
```

Ideally we should run tests in isolation.